api_app/playbooks_manager/dataclasses.py

```diff
@@ -0,0 +1,57 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import dataclasses
+import typing
+
+from .serializers import PlaybookConfigSerializer
+
+__all__ = ["PlaybookConfig"]
+
+# Try to see if your changes in serializers helps with the changeshere (make them)
+
+
+@dataclasses.dataclass
+class PlaybookConfig:
+    name: str
+    description: str
+    supports: typing.List[str]
+    disabled: bool
+    analyzers: typing.Dict[str, typing.Any]
+    connectors: typing.Dict[str, typing.Any]
+
+    serializer_class = PlaybookConfigSerializer
+
+    @property
+    def is_ready_to_use(self) -> bool:
+        return not self.disabled
+
+    @classmethod
+    def from_dict(cls, data: dict) -> "PlaybookConfig":
+        keys_to_pop = ["config", "verification", "secrets", "params", "python_module"]
+        [data.pop(key) for key in keys_to_pop]
```

Comment:
I'm ok with having this utility method, but popping wrong keys makes the assumption that `data` is formatted in some way. You should, instead, follow the opposite approach: whitelist only the keys that should be parsed.

Reply:
Way better approach! I will look into this. The code didn't require popping until the last version release. After I took care of the conflicts, This issue had to be quickly put to rest like this.